src/connection/activate.ts

```diff
@@ -32,7 +32,7 @@ function useTryActivation() {
   const currentPage = getCurrentPageFromLocation(pathname)
 
   return useCallback(
-    async (connection: Connection, onSuccess: () => void, chainId?: SupportedChainId) => {
+    async (connection: Connection, onSuccess?: () => void, chainId?: SupportedChainId) => {
```

Comment:
I needed to change this to get the test to pass. If we're not happy with this we can try to fix the test instead?

Reply:
which test?